### ISSUE TITLE
No explicit docs on using Openport with a key

### ISSUE THREAD

**Issue by detly** [openportio/openport-go#1]
Looking at the [usage docs](https://openport.readthedocs.io/), there is nothing mentioned about using Openport with a key obtained from a paid account.

The usage text mentions a `register` subcommand, but no info is given on it.

On one of the recipes, there is a `--register-key` option used. This option is not mentioned in the help text, nor does it match the `register` subcommand.

If these instructions are gated behind the user accounts page, that's not helpful in situations where a different person in an organisation may be setting up the account vs. integrating it into a device.

**Comment by jandebleser** [comment]
Indeed, the "register" and "--register-key" commands should be "register-key". I have updated the docs, and added a section about registering the key on the usage page. https://openport.readthedocs.io/en/latest/usage.html#linking-a-key-to-your-account
Thanks for the remark.
